core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts

```diff
@@ -136,6 +136,8 @@ export class DotContainerPropertiesComponent implements OnInit {
         const formValues = this.form.value;
         if (formValues.identifier) {
             this.store.editContainer(formValues);
+            this.store.updateOriginalFormState(formValues);
+            this.form.updateValueAndValidity();
```

Comment:
test to check these functions are doing any changes!

Reply:
Noted

Reply:
test case added.